test/components/dropdown/dropdown.test.js

```diff
@@ -6,9 +6,13 @@ import Dropdown from '../../../src/components/dropdown/dropdown';
 describe('<Dropdown />', () => {
   let wrapper;
 
-  beforeEach(() => wrapper = shallow(
-    <Dropdown toggle="Dropdown" actions={[{ label: 'log', action() {console.log('I was clicked!');} }]} />
-  ));
+  beforeEach(() => {
+    wrapper = shallow(
+      <Dropdown
+        toggle="Dropdown"
+        actions={ [{ label: 'log', action() { console.log('I was clicked!'); } }] }
```

Comment:
Don't use `console.log()` in tests. It's an external dependency that should not be present in unit tests. Either supply a spy if you need to verify that action was triggered or use empty function.